Units/parser-python.r/python3-function-annotations.d/expected.tags

```diff
@@ -1,12 +1,12 @@
-a	input.py	/^def func03(a:"hello", b : 1 * 2 + 3 \/ 4 = 5):$/;"	z	function:func03	file:
-a	input.py	/^def func04(a:(1, 2, 3, 4)=(1, 2, 3, 4), b:[5,6]=[7,8], c=0) -> {'foo' : 'bar' }:$/;"	z	function:func04	file:
-b	input.py	/^def func03(a:"hello", b : 1 * 2 + 3 \/ 4 = 5):$/;"	z	function:func03	file:
-b	input.py	/^def func04(a:(1, 2, 3, 4)=(1, 2, 3, 4), b:[5,6]=[7,8], c=0) -> {'foo' : 'bar' }:$/;"	z	function:func04	file:
+a	input.py	/^def func03(a:"hello", b : 1 * 2 + 3 \/ 4 = 5):$/;"	z	function:func03	typeref:typename:"hello"	file:
```

Comment:
I don't think you can use a literal/expression as the type hint. See https://github.com/universal-ctags/ctags/issues/2606#issuecomment-670047745. Does it work for your toolchain?

Reply:
`python` command itself accepts (and ignores) this broken hint.

Reply:
As far as I know, type hint is meant to work with tools like `mypy`. Python itself doesn't bother with it. So the code in this test definitely works well as a test, but it doesn't look like a real python file that utilizes type hint.

Reply:
> but it doesn't look like a real python file that utilizes type hint.

Yes, but users can give broken input to ctags.  ctags should not crash when broken input is given.
This test input tests this aspect.

Reply:
Ok, I got it.